### PR TITLE
Some minor fixes

### PR DESCRIPTION
**Pull request by norhh**
- Checks whether call object is None or not in delegatecall module
- Converts python bool to z3 bool to support `Or()` in sstore.

### CODE REVIEW

File: mythril/laser/ethereum/instructions.py [hunk 1]
```diff
@@ -1004,7 +1004,7 @@ def sstore_(self, global_state: GlobalState) -> List[GlobalState]:
             keccak_keys = filter(keccak_function_manager.is_keccak, storage_keys)
 
             results = []
-            new = False
+            new = Bool(False)
```
Comment (JoranHonig) [inline mythril/laser/ethereum/instructions.py]: We should use the symbol factory for this. But it doesn't have a method for Bool yet, could you add that?

Comment (norhh) [reply mythril/laser/ethereum/instructions.py]: Done

File: mythril/laser/smt/__init__.py [hunk 2]
```diff
@@ -56,6 +66,17 @@ class _SmtSymbolFactory(SymbolFactory):
...
+    @staticmethod
+    def Bool(value: bool, annotations=None):
+        """
+        Creates a Bool with concrete value
+        :param value: The boolean value
+        :param annotations: The annotations to initialize the bool with
+        :return: The freshly created Bool()
+        """
+        raw = z3.Bool(value)
+        return Bool(raw)
```
Comment (JoranHonig) [inline mythril/laser/smt/__init__.py]: the annotations need to be passed as a parameter right?

Comment (norhh) [reply mythril/laser/smt/__init__.py]: argh!

**Pull request merged**